src/main/webapp/WEB-INF/style/main.css

```diff
@@ -0,0 +1,56 @@
+* {
+    padding: 0;
+    margin: 0;
+    font-family: "Arial Black";
+}
+body {
+    background: black;
+}
+.content {
+    padding-top: 10vh;
+    width: 80vw;
+    margin: 0 auto;
+}
+h1 {
+    font-size: 3em;
+    color: white;
+    margin-bottom: 5vh;
+}
+
+table {
+    margin-top: 5vh;
+}
+p {
+    font-size: 2em;
+    color: white;
+}
+
+td, tr {
+    font-size: 1.5em;
+    color: white;
+    padding: 10px;
+}
+
+form {
+    margin-top: 10vh;
+}
+
+input, button {
+    padding: 10px;
+    font-size: 2em;
+}
+
+button {
+    margin-top: 6vh;
+}
+
+p {
+    margin-top: 5vh;
+}
+
+a {
+    margin-top: 10vh;
+    font-size: 2.5em;
+    font-family: "Arial Black";
+    text-decoration: none;
+}
```

Comment:
add a new line at the end of all JSP files and CSS

Reply:
added )